routes/web.php

```diff
@@ -93,7 +93,8 @@ function (): void {
             Route::middleware(['auth'])->group(function (): void {
 
                 /* Managers */
-                Route::get('managers/{manager}', 'ManagerProfileController@show')
+                Route::get('jobs/{jobPoster}/manager/{manager}', 'ManagerProfileController@show')
```

Comment:
If we're using the job id in the url, we actually don't need the manager id anymore, since each job has a unique manager.